New version: AstroPlot v0.1.2

JuliaRegistrator opened the pull request:
- Registering package: AstroPlot
- Repository: https://github.com/JuliaAstroSim/AstroPlot.jl
- Created by: @islent
- Version: v0.1.2
- Commit: b78b4e551b881ccdf2c8d47ceb0a60f08e93026c
- Reviewed by: @islent
- Reference: https://github.com/JuliaAstroSim/AstroPlot.jl/commit/b78b4e551b881ccdf2c8d47ceb0a60f08e93026c#commitcomment-138207602
- Description: Astrophysical plotting interfaces for Julia
- Release notes:
<!-- BEGIN RELEASE NOTES -->
`````
- adapt Makie API
- update compat
- precompile with PrecompileTools.jl
`````
<!-- END RELEASE NOTES -->

<!-- bf0c69308befbd3ccf2cc956ac8a46712550b79fc9bfb5e4edf8f833f05f4c18b06eddad8845b45beb9f45c2b8020dd68fcf0ffa4e7c38f1fc70ec48d127f851112f638b913230f7d0d1393f7adc466f26bb75a7d75e26fa34d3efdcf5d6ed4f0e517a3bfde577995318982b05457b95e77d4a503913d3c320f4a03ec2e784f84045819d26209acee411c8fc8c829f175cd6095c77ab142d4d7c4cdc4cac8e8118bb2c7d0ed540ca279ac43aa3b20e6521483319ef5f69009710c909d8c356ea255de96371ec265ea18ed03e28229339dc3a6d2b51c8f1dc435152a63a33fc2d -->

islent commented:
> import AstroPlot failed). See the AutoMerge logs for details

Sorry that I cannot find any log related to `import AstroPlot`. There is no log of `Automerge` process. Need help!!!

islent commented:
Oh hold on, I found it in the complete check lists:
https://github.com/JuliaRegistries/General/actions/runs/7756160398/job/21152936390

JuliaRegistrator committed:
New version: AstroPlot v0.1.2

UUID: 00b67d96-df8a-4e04-ab4d-0170e12eeaec
Repo: https://github.com/JuliaAstroSim/AstroPlot.jl.git
Tree: 5cc936e06108bd46a0e12ad9ddbb2490ec87db6c

Registrator tree SHA: 17aec322677d9b81cdd6b9b9236b09a3f1374c6a
